addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -259,12 +259,11 @@ snippetsEditor.Class.include({
     /**
      * @override
      */
-    _insertDropzone: function ($hook) {
-        var $hookParent = $hook.parent();
-        var $dropzone = this._super(...arguments);
-        $dropzone.attr('data-editor-message', $hookParent.attr('data-editor-message'));
-        $dropzone.attr('data-editor-sub-message', $hookParent.attr('data-editor-sub-message'));
-        return $dropzone;
+    _setDropZone: function ($drop) {
```

Comment:
@bvr-odoo You need to adapt to the new improvements that were made in 13.0, not force your work over it :wink: You are in a stable version, you cannot rename functions like that nor change their purpose. Custo may rely on them. This function should still be in charge of creating the dropzone and so no more `zoneTemplate.clone()` stuff regressing to the old code of 12.0.